Sources/Utils/Rng/MobileCoinChaCha20Rng.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+import LibMobileCoin
+
+final class MobileCoinChaCha20Rng: MobileCoinSeedableRng {
```

Comment:
looking at these FFI calls - https://github.com/mobilecoinfoundation/mobilecoin/blob/f3eff68f1c8bf53e8635c63b10bc79197e4b1848/libmobilecoin/src/chacha20_rng.rs - the `unwrap()`s & `expect()`s will cause rust panics. The pattern for dealing with errors can be seen in other rust files, and their corresponding swift wrappers. Change the FFI calls, and these call-sites to catch and "re-throw" if needed.